FlowCryptTests/Functionallity/WKDURLs/WKDURLsConstructorTests.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  WKDURLsTests.swift
+//  FlowCryptTests
+//
+//  Created by Yevhen Kyivskyi on 17.05.2021.
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import XCTest
+
+class WKDURLsTests: XCTestCase {
+    
+    var constructor: WKDURLsConstructorType!
+
+    override func setUp() {
+        constructor = WKDURLsConstructor()
+    }
+
+    override func tearDown() {
+        constructor = nil
+    }
+
+    func test_direct_mode_construct_URL_success() {
```

Comment:
Please also test uppercase emails - direct and advanced - as per issue (the method must lowercase the email before hashing it, etc).